src/main/data/languages.json

```diff
@@ -67,624 +55,525 @@
   },
   {
     "comments": [
-      "DCNC notes: (Audio only) - Chinese - Mandarin PRC"
+      "(Audio only)"
     ],
     "dcncTag": "CMN",
+    "language": "Chinese - Mandarin PRC",
     "rfc5646Tag": "cmn",
     "use": [
       "audio"
     ]
   },
   {
     "comments": [
-      "DCNC notes: (Subtitles only) - Chinese - Mandarin Simplified"
+      "(Subtitles only)"
     ],
     "dcncTag": "QMS",
+    "language": "Chinese - Mandarin Simplified",
     "rfc5646Tag": "cmn-Hans",
     "use": [
       "text"
     ]
   },
   {
     "comments": [
-      "DCNC notes: (Subtitles only) - Chinese - Mandarin Traditional"
+      "(Subtitles only)"
     ],
     "dcncTag": "QMT",
+    "language": "Chinese - Mandarin Traditional",
     "rfc5646Tag": "cmn-Hant",
     "use": [
       "text"
     ]
   },
   {
     "comments": [
-      "DCNC notes: (Audio only) - Chinese - Taiwanese"
+      "(Audio only)"
     ],
     "dcncTag": "NAN",
+    "language": "Chinese - Taiwanese",
     "rfc5646Tag": "nan",
     "use": [
       "audio"
     ]
   },
   {
     "comments": [
-      "DCNC notes: (Audio only) - Chinese - Taiwanese Mandarin"
+      "(Audio only)"
```

Comment:
This can be removed since `use` is `audio` only.

Reply:
Removed all audio only in notes where applicable, left subtitles only notes.